runtime/compiler/optimizer/EscapeAnalysis.hpp

```diff
@@ -135,8 +135,8 @@ class Candidate : public TR_Link<Candidate>
     	_virtualCallSitesToBeFixed(c->trMemory()),
         _coldBlockEscapeInfo(c->trMemory())
          {
-          static const char *forceContinguousAllocation = feGetEnv("TR_forceContinguousAllocation");
-          if (forceContinguousAllocation)
+          static const char *forceContiguousAllocation = feGetEnv("TR_forceContiguousAllocation");
```

Comment:
Probably don't want to change the environment variable now.
@andrewcraik

Reply:
Tentatively will drop (but probably not before Tuesday).